SwiftBasicsExercise.playground/Pages/Dictionaries.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,227 @@
+"c" : "d",
+"d" : "e",
+"e" : "f",
+"f" : "g",
+"g" : "h",
+"h" : "i",
+"i" : "j",
+"j" : "k",
+"k" : "l",
+"l" : "m",
+"m" : "n",
+"n" : "o",
+"o" : "p",
+"p" : "q",
+"q" : "r",
+"r" : "s",
+"s" : "t",
+"t" : "u",
+"u" : "v",
+"v" : "w",
+"w" : "x",
+"x" : "y",
+"y" : "z",
+"z" : "a"
+]
+
+var encodedMessage = "uijt nfttbhf jt ibse up sfbe"
+var decodedMessage = ""
+for charr in encodedMessage{
+    if String(charr) == " "{
+        decodedMessage+=" "
+    }
+    for (key,value) in code {
+        if String(charr) == value{
+            decodedMessage+=key
+        }
+    }
+}
+print(decodedMessage)
+
+//2
+var people: [[String:String]] = [
+[
+"firstName": "Calvin",
+"lastName": "Newton"
+],
+[
+"firstName": "Garry",
+"lastName": "Mckenzie"
+],
+[
+"firstName": "Leah",
+"lastName": "Rivera"
+],
+[
+"firstName": "Sonja",
+"lastName": "Moreno"
+],
+[
+"firstName": "Noel",
+"lastName": "Bowen"
+]
+]
+
+var firstNames : [String] = []
+for name in people {
+    firstNames.append(name["firstName"]!)
+}
+print(firstNames)
+
+//3
+var people: [[String:Any]] = [
+[
+"firstName": "Calvin",
+"lastName": "Newton",
+"score": 13
+],
+[
+"firstName": "Garry",
+"lastName": "Mckenzie",
+"score": 12
+],
+[
+"firstName": "Leah",
+"lastName": "Rivera",
+"score": 10
+],
+[
+"firstName": "Sonja",
+"lastName": "Moreno",
+"score": 3
+],
+[
+"firstName": "Noel",
+"lastName": "Bowen",
+"score": 16
+]
+]
+var fullNames : [String] = []
+
+for name in people {
+    if let fName = name["firstName"] as? String {
+        if let lName = name["lastName"] as? String {
+            var fullName = fName + " " + lName
+            fullNames.append(fullName)
+        }
+    }
+}
+print(fullNames)
+
+//4
+var people: [[String:Any]] = [
+[
+"firstName": "Calvin",
+"lastName": "Newton",
+"score": 13
+],
+[
+"firstName": "Garry",
+"lastName": "Mckenzie",
+"score": 12
+],
+[
+"firstName": "Leah",
+"lastName": "Rivera",
+"score": 10
+],
+[
+"firstName": "Sonja",
+"lastName": "Moreno",
+"score": 3
+],
+[
+"firstName": "Noel",
+"lastName": "Bowen",
+"score": 16
+]
+]
+var name = ""
+if let minScore = people[0]["score"] as? Int {
+    var ind = 0
+    for person in people {
+        if let score = person["score"] as? Int{
+            if score < minScore{
+                if let fName = person["firstName"] as? String {
+                        if let lName = person["lastName"] as? String {
+                            var fullName = fName + " " + lName
+                            name = fullName
+                        }
+                    }
+            }
+        }
+    }
+}
+print(name)
+
+//5
+var people: [[String:Any]] = [
+[
+"firstName": "Calvin",
+"lastName": "Newton",
+"score": 13
+],
+[
+"firstName": "Garry",
+"lastName": "Mckenzie",
+"score": 23
+],
+[
+"firstName": "Leah",
+"lastName": "Rivera",
+"score": 10
+],
+[
+"firstName": "Sonja",
+"lastName": "Moreno",
+"score": 3
+],
+[
+"firstName": "Noel",
+"lastName": "Bowen",
+"score": 16
+]
+]
+
+let so = people.sorted {
```

Comment:
Don't use shorthand names for varibales, make them expressive as much as possible